Application/EdFi.Ods.SandboxAdmin/wwwroot/js/edfiadmin.auth.js

```diff
@@ -1,6 +1,6 @@
 ﻿/// <reference path="globals.js" />
 /// <reference path="extensions.js" />
-/// <reference path="underscore.js" />
+/// <reference path="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.5.2/underscore-min.js" />
```

Comment:
does this do anything? my understanding was that reference paths no longer work in newer version of Visual Studio

Reply:
removed reference paths